docs/api/rasa-sdk.rst

```diff
@@ -0,0 +1,145 @@
+:desc: Extend your Rasa conversational AI assistant using Rasa-SDK to connect to
+       external APIs or improve dialogue with custom actions written in Python.
+
+.. _rasa-sdk:
+
+Rasa SDK
+========
+
+.. edit-link::
+
+Rasa SDK provides the tools you need to write custom actions in python.
+
+.. contents::
+   :local:
+
+Installation
+------------
+
+Use ``pip`` to install ``rasa-sdk`` on your action server.
+
+.. code-block:: bash
+
+    pip install rasa-sdk
+
+.. note::
+
+    You do not need to install ``rasa`` for your action server.
+    E.g. if you are running Rasa in a docker container, it is recommended to
+    create a separate container for your action server. In this
+    separate container, you only need to install ``rasa-sdk``.
+
+Running the Action Server
+-------------------------
+
+If you have ``rasa`` installed, run this command to start your action server:
+
+.. code-block:: bash
+
+    rasa run actions
+
+.. _custom_action_example:
```

Comment:
why is this here?

Reply:
the docs would be very different had you been here from the beginning to review them xP

Reply:
Hahahaha